src/main/java/com/alkemy/ong/data/gateways/DefaultMemberGateway.java

```diff
@@ -0,0 +1,76 @@
+package com.alkemy.ong.data.gateways;
+
+import com.alkemy.ong.data.entity.MemberEntity;
+import com.alkemy.ong.data.repository.MemberRepository;
+import com.alkemy.ong.domain.members.Member;
+import com.alkemy.ong.domain.members.MemberGateway;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class DefaultMemberGateway implements MemberGateway {
+
+    private final MemberRepository memberRepository;
+
+    @Autowired
+    public DefaultMemberGateway(MemberRepository memberRepository) {
+        this.memberRepository = memberRepository;
+    }
+
+    @Override
+    public Member create(Member model) {
+        MemberEntity entity = this.toEntity(model);
+        MemberEntity entitySaved = this.memberRepository.save(entity);
+        return this.toModel(entitySaved);
+    }
+
+    @Override
+    public List<Member> findAll() {
+        List<MemberEntity> entityList = this.memberRepository.findAll();
+        return this.toModelList(entityList);
```

Comment:
en lugar de resolverlo con un for-each usemos streams q son un perfect fit para esto.
La idea es:
members.stream().map(funcion q se aplica a cada elemento d la lista).collect(toList());